### PR TITLE
tests: fixed tests and flow

### PR DESCRIPTION
**Pull request by souhe**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -73,12 +73,13 @@
     "create-react-context": "^0.2.1",
     "deepmerge": "^2.1.1",
     "flow-copy-source": "^2.0.2",
-    "hoist-non-react-statics": "^2.5.0",
+    "hoist-non-react-statics": "2.5.0",
```
Comment (satya164) [inline package.json]: Can we not use the exact version? It'll prevent de-duplication by yarn/npm

File: package.json [hunk 2]
```diff
@@ -73,12 +73,13 @@
...
     "prop-types": "^15.6.0"
   },
   "jest": {
     "moduleNameMapper": {
       "\\.(css|less)$": "<rootDir>/__mocks__/styleMock.js"
-    }
+    },
+    "testURL": "http://localhost/"
```
Comment (satya164) [inline package.json]: 🤔

File: package.json (hunk 1 above)
Comment (souhe) [reply package.json]: I tried but it didn't work. I still have an error "required module not found". So I decided to force 2.5.0 now to be able to release new version with other fixes

File: package.json (hunk 2 above)
Comment (souhe) [reply package.json]: without it tests don't work.

Comment (souhe) [reply package.json]: https://github.com/jsdom/jsdom/issues/2304

Comment (souhe) [reply package.json]: https://github.com/jsdom/jsdom/issues/2304

**Pull request closed without merging**